cmd/scaler/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM prombench/prombench:2.0.0
+FROM alpine:3.8
 
-COPY scaler /prombench/scaler
-COPY Makefile /prombench/Makefile
+COPY scaler scaler
+COPY tmp/2_fake-webserver.yaml 2_fake-webserver.yaml
 
-WORKDIR /prombench
\ No newline at end of file
+ENTRYPOINT ["/scaler", "scale","-f", "/2_fake-webserver.yaml"]
```

Comment:
hm , this doesn't look right. The entrypoint should only include the binary name. any args should be passed at runtime.


We need to find a way to mount any config or manifest files at runtime instead of adding them to each image at build-time.